### DISCUSSION TITLE
Get TV Pointer Position

### DISCUSSION THREAD

**Discussion by iceFbr** [supersaiyanmode/PyWebOSTV#66] [General]
Hi, just a question: 
Is there any way to retrieve pointer position from webos? 

I am trying to hook it in order to control my pc mouse with magic remote =)

ty!!!

**Comment by supersaiyanmode** [comment]
I am not aware of any API (over the dedicated WebSocket channel or otherwise) that lets you get the cursor position. That being said, I really like your idea.  I remember reading somewhere that if you project your screen over wireless display, the magic remote works as a mouse of the computer.

**Reply by iceFbr** [reply]
well, i just bouth an cheap air remote control and plug it in my desktop. 
I tried every way to retrive the pointer location from tv but failed... 
Ty! If you have any insights on this matter plz tell me =)

**Comment by Fire-** [comment]
@supersaiyanmode are we able to use the luna magic remote APIs[^1]? ( I don't quite understand the difference in the ssap and luna api's )

[^1]: https://webostv.developer.lge.com/api/webos-service-api/magic-remote

**Reply by supersaiyanmode** [reply]
To be honest, I don't either :). There's a chance it could work -- I've been meaning to try it out for quite some time now, but I have been a little busy with other stuff.

Could someone (perhaps you?) here volunteer to try it out? Essentially, a new dummy class extending `WebOSControlBase` (replicate `SystemControl`), change the URI field to the value in Luna APIs, instantiate an object and see if it works. If it does, we have opened up a brand new set of APIs that users of this library can use :)
